pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -420,6 +422,16 @@ func (configgen *ConfigGeneratorImpl) buildSidecarInboundListeners(env *model.En
 				Name:     ingressListener.Port.Name,
 			}
 
+			// if app doesn't have a declared ServicePort, but a sidecar ingress is defined - we can't generate a listener
+			// for that port since we don't know what policies or configs apply to it ( many are based on service matching).
+			// Sidecar doesn't include all the info needed to configure a port.
+			instance := configgen.findInstance(proxyInstances, ingressListener)
```

Comment:
See earlier comment. This logic will preclude all UDS bound listeners unless you match by port name

Reply:
Agreed, but can be solved in separate PRs.

Reply:
I mean - it's not a regression introduced by this PR, and this PR is not testing UDS.